### PR TITLE
Separate plugin installation process by os

### PR DESCRIPTION
**Pull request by asukiaaa**
I'm a linux user and it took time to show plugin list.
I thins the information is not optional so I separate process by os.

Thank you.

### CODE REVIEW

**Comment by asukiaaa** [comment]
@googlebot I signed it!

File: src/docs/get-started/editor.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Alternatively, you can also use IntelliJ:
 
 ## Install the Flutter and Dart plugins
 
-To install these:
+### Mac
```
Comment (sfshaza2) [inline src/docs/get-started/editor.md:54]:
```suggestion
The installation instructions vary by platform.

### Mac

Use the following instructions for macos:
```

File: src/docs/get-started/editor.md [hunk 2]
```diff
@@ -61,13 +61,11 @@ To install these:
   1. Click **Yes** when prompted to install the Dart plugin.
   1. Click **Restart** when prompted.
 
-{{site.alert.note}}
-  Prior to v3.6.3.0, access plugin preferences as follows:
-   1. Open plugin preferences (on MacOS go to **Preferences > Plugins**; 
-      on Windows & Linux go to **File > Settings > Plugins**).
+### Linux or Windows
+
+   1. Open plugin preferences (**File > Settings > Plugins**).
```
Comment (sfshaza2) [inline src/docs/get-started/editor.md:70]:
```suggestion
### Linux or Windows

Use the following instructions for Linux or WIndows:

   1. Open plugin preferences (**File > Settings > Plugins**).
```

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**